src/components/common/TxModalDialog/index.tsx

```diff
@@ -25,8 +25,7 @@ const TxModalDialog = ({
       scroll={fullScreen ? 'paper' : 'body'}
       className={classnames(css.dialog, { [css.fullWidth]: fullWidth })}
       onClick={(e) => e.stopPropagation()}
-      TransitionComponent={Fragment}
```

Comment:
If we don't set the `TransitionComponent`, the modal does not immediately mount. It's not an issue but thought I'd make you aware.